### DISCUSSION TITLE
Sunbird RC - resources are not loading in local installation

### DISCUSSION THREAD

**Discussion by Kovendan03** [Sunbird-RC/community#274] [General]
Hi Team,

I am trying to set-up Sunbird RC locally by following the instructions - https://github.com/code-blooded-guy/sunbird-rc-ui/blob/main/README.md,  able to install successfully. But when i try to click School/Teacher/Student, getting error - something went wrong with schema url or path.  Config.json are available in the path. I tried copying forms.json from demo-education-registry/schemas repository but still the same error. Page gets stuck in loading.

![image](https://user-images.githubusercontent.com/114154088/193515902-959e91ef-fafe-404d-8fd7-2b055dfe82b4.png)

**Comment by beepdot** [comment]
@reshmi-nair @amit-tarento Can you please check and respond? Or please tag the right person who can answer this.

**Reply by parthlawate** [reply]
@Pratikshakhandagale @code-blooded-guy Can you help with this ? 

cc @Renuka-S

**Reply by reshmi-nair** [reply]
Tagging @tejash-jl and @dileepbapat

**Comment by Pratikshakhandagale** [comment]
@keshavprasadms Can you please go to sunbird-rc-ui/src/assets/config path and check ui-config folder is present or not. 

If not, please confirmed that you have  followed the step 3 and 4 correctly from here - https://github.com/code-blooded-guy/sunbird-rc-ui/blob/main/README.md. 

or if its present, go into that folder and share header.json file with us.

**Reply by beepdot** [reply]
@Kovendan03 The above message is addressed to you. Please check and update back to @Pratikshakhandagale

**Comment by Kovendan03** [comment]
@keshavprasadms 

Thanks for your reply. I have followed the steps 3 & 4 mentioned in the document. Below are the list of folders.

~/sunbird-rc-ui/src/assets/config# ls -l
total 52
-rw-r--r-- 1 root root  1082 Sep 30 14:47 config.json
-rw-r--r-- 1 root root 45023 Oct  3 06:04 forms.json
-rw-r--r-- 1 root root  2637 Sep 29 05:05 home.html
lrwxrwxrwx 1 root root    39 Sep 29 11:43 ui-config -> /root/demo-education-registry/ui-config

~/sunbird-rc-ui/src/assets/config/ui-config# ls -l
total 68
-rw-r--r-- 1 root root 45023 Sep 29 05:05 forms.json
-rw-r--r-- 1 root root  2633 Sep 29 05:05 headers.json
-rw-r--r-- 1 root root  5650 Sep 29 05:05 layouts.json
-rw-r--r-- 1 root root  5138 Sep 29 05:05 search.json
-rw-r--r-- 1 root root  2622 Sep 29 05:05 tables.json

Pasting  "header.json" for your reference.

{
    "type": "opensaberLayoutSchema",
    "version": "0.1",
    "headers": [
        {
            "default":{
                "right":[
                    {
                        "title": "LOGIN",
                        "redirectTo": "/login"
                    }
                ]
            }
        },
        {
            "institute": {
                "left": [
                    {
                        "title": "HOME",
                        "redirectTo": "/profile/institute"
                    },
                    {
                        "title": "ATTESTATIONS",
                        "redirectTo": "/institute/attestation/institute-attestation"
                    }
                ],
                "right": [
                    {
                        "title": "LOGOUT",
                        "redirectTo": "/logout",
                        "className" : "font-weight-bold"
                    }
                ]
            }
        },
        {
            "teacher": {
                "left": [
                    {
                        "title": "HOME",
                        "redirectTo": "/profile/teacher"
                    },
                    {
                        "title": "ATTESTATIONS",
                        "redirectTo": "/teacher/attestation/teacher-attestation"
                    }
                ],
                "right": [
                    {
                        "title": "LOGOUT",
                        "redirectTo": "/logout",
                        "className" : "font-weight-bold"
                    }
                ]
            }
        },
        {
            "student": {
                "left": [
                    {
                        "title": "HOME",
                        "redirectTo": "/profile/student"
                    }
                ],
                "right": [
                    {
                        "title": "LOGOUT",
                        "redirectTo": "/logout",
                        "className" : "font-weight-bold"
                    }
                ]
            }
        },
        {
            "board-cbse": {
                "left": [
                    {
                        "title": "ATTESTATIONS",
                        "redirectTo": "/profile/board-cbse"
                    }
                ],
                "right": [
                    {
                        "title": "LOGOUT",
                        "redirectTo": "/logout",
                        "className" : "font-weight-bold"
                    }
                ]
            }
        }
    ]
}

**Reply by sagivpn** [reply]
@Pratikshakhandagale Request you to check the above message by @Kovendan03

**Reply by Pratikshakhandagale** [reply]
@Kovendan03 Seems header.json file is correct. Can you please join the[ discord channel ](https://discord.gg/Q5mvw2mGC8)and ping me. We can join a call and check this issue.

**Reply by Kovendan03** [reply]
@Pratikshakhandagale please let me know the time, i will join the discord channel today. Thanks.

**Comment by Kovendan03** [comment]
Installed RC backend components but still having issues.

**Reply by tejash-jl** [reply]
can you share the issues you're facing and also the logs for it.

**Comment by Kovendan03** [comment]
able to install the backend services, but still frontend is having issues.

**Reply by parthlawate** [reply]
@Pratikshakhandagale pls help cc @code-blooded-guy

**Reply by Pratikshakhandagale** [reply]
@Kovendan03 Can you please ping me on discot channel. We can join a call and check this issue.

**Reply by Kovendan03** [reply]
@Pratikshakhandagale, I will connect with you today to resolve the issue.

**Reply by parthlawate** [reply]
was this solved @Kovendan03 @Pratikshakhandagale ?
